src/sass/components/_repo-images.scss

```diff
@@ -0,0 +1,42 @@
+// Repo Images
+
+// Variables specific to repo images
+$repo-img-size: calc(3.5rem + 3vw);
+$repo-img-size-static: 4.75rem;
+$repo-img-pos-adjust: -1.125rem;
+
+.ms-repo-item {
+  // increase default height/weight of icon in repo visualpicker
+  .slds-media__figure {
+    > div {
+      height: $repo-img-size;
+      width: $repo-img-size;
+
+      .slds-avatar_large {
+        height: 100%;
+        width: 100%;
+        font-size: $font-size-10;
+        font-weight: $font-weight-bold;
+      }
+
+      abbr {
+        text-shadow: none;
+      }
+    }
+  }
+}
+
+// When image, name and description are displayed on project list sidebar
+.ms-repo-image-wrapper {
+  height: $repo-img-size;
+  width: $repo-img-size;
+}
```

Comment:
Consider using a single selector block to change the avatar size based on either wrapper? Rather than repeating this height/width in different places?

Reply:
This variable is used in two places: Sidebar and List ... because when used in the repo list (homepage) we target an element we can't add a class to (one of those divs that doesn't have a class) .... or is that not what you meant? These two locations aren't nested the same either. Let me know if that isn't what you had in mind.

The other variable for sizing (static size) is used in the page header.

Reply:
ah, this makes more sense now that I see we don't always have `slds-avatar` classes available. Looks good!